ms2deepscore/train_new_model/ValidationLossCalculator.py

```diff
@@ -14,14 +14,9 @@
 class ValidationLossCalculator:
     def __init__(self,
                  val_spectrums,
-                 settings: SettingsMS2Deepscore,
-                 score_bins=None,
-                 random_seed=42):
-        self.val_spectrums = select_one_spectrum_per_inchikey(val_spectrums, random_seed)
-        if score_bins is None:  # Use same as for training
-            self.score_bins = settings.same_prob_bins
-        else:
-            self.score_bins = score_bins
+                 settings: SettingsMS2Deepscore):
```

Comment:
I think we would always want to use the same_prob_bins that were used during training here. So to reduce the risk of introducing errors, this might be best to remove as an option. (if different prob_bins, val loss cannot be compared to train loss)

Reply:
Thats fine with me , too!